README.md

```diff
@@ -1,181 +1,78 @@
-# Tracee
-Container and system tracing using eBPF
+# Tracee - Container and system tracing using eBPF
 
-**Tracee** is a lightweight, easy to use container and system tracing tool.
-After launching the tool, it will start collecting traces of newly created containers (container mode) or processes (system mode).
-The collected traces are mostly system calls performed by the processes,
-but other events, such as capabilities required to perform the actions requested by the process, are also supported.
+Tracee is a lightweight and easy to use container and system tracing tool. It allows you to observe system calls and other system events in real time. A unique feature of Tracee is that it will only trace newly created processes and containers (that was started after Tracee has started), in order to help the user focus on relevant events instead of every single thing that happens on the system (which can be overwhelming).
 
-Tracee CLI was originally written in Python, but is now being ported to Go. Currently both versions are available in the repo. Python is considered more stable and tested, but future development will be in Go. This readme is referring to the Python version as it's recommended for newcomers but all of the features are similarly supported in the Go version, give or take minor CLI differences.
+Tracee CLI was originally written in Python, but was since ported to Go. Currently both versions are still available in the repo, but future development will be in Go and the Python version will eventually be deprecated and removed.
 
-## Requirements
-Currently requires 
-* kernel version > 4.14
-* BCC
-* Python if using the Python version
+## Getting started
 
-## Quick Start Instructions
+### Prerequisites
+To run, Tracee requires the following:
+* Linux kernel version > 4.14
+* Kernel headers
+* C standard library (currently tested with glibc)
+* [BCC](https://github.com/iovisor/bcc)
```

Comment:
Maybe add "For convenience we provide a containerised version of Tracee that includes the kernel headers, glibc and BCC requirements that should run on any kernel > 4.14", wdyt?

Reply:
the image does not include the kernel headers, but I'll add the note without it